### ISSUE TITLE
[Accessibility] Visual label should be defined for the combo box request method option.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1863]
**Test Environment:**
OS Build: Version 22H2(OS Build 25140.1000) 
Browser: Version 104.0.1278.2 (Official build) dev (64-bit)
URL: [Graph Explorer | Try Microsoft Graph APIs - Microsoft Graph](https://developer.microsoft.com/en-us/graph/graph-explorer)
Screen reader: Narrator

**To Reproduce**

1. Open the above [URL](https://developer.microsoft.com/en-us/graph/graph-explorer) in Edge browser.
2. Navigate the to request method option combo box.
3. Observe the issue.

**Actual Result:**
Visual label is not defined for the combo box request method option.

**Expected Result:**
Visual label should be defined for the combo box request method option

**User Impact:**
The Keyboard users will not know about the visual text or the understand the purpose of the combo box control.

**MAS Reference:**
[Name, Role, Value - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/04.01.02)

**Additional context**
Issue: https://office.visualstudio.com/OC/_workitems/edit/6175384/

**Comment by ddyett** [comment]
waiting on design feedback

**Comment by ElinorW** [comment]
Closing this as it has been resolved as 'by design'
https://office.visualstudio.com/OC/_workitems/edit/6175384?src=WorkItemMention&src-action=artifact_link

Should there be a design change requested, this shall be tracked on a new ticket.
